lib/Extractor/KLEEBuilder.cpp

```diff
@@ -136,7 +140,23 @@ ref<Expr> ExprBuilder::makeSizedArrayRead(unsigned Width, StringRef Name,
       new Array(ArrayNames.makeName(NameStr), 1, 0, 0, Expr::Int32, Width));
   ArrayVars.push_back(Origin);
 
+  std::vector<unsigned> ZeroBits, OneBits;
   UpdateList UL(Arrays.back().get(), 0);
+  if (HarvestKnownBits && Origin && Origin->K == Inst::Var) {
+    ref<Expr> Result1 = klee::ConstantExpr::create(0, Expr::Bool);
+    ref<Expr> Result2 = klee::ConstantExpr::create(0, Expr::Bool);
+    ref<Expr> Var = ReadExpr::create(UL, klee::ConstantExpr::alloc(0, Expr::Int32));
+    // KnownZeros
+    ref<Expr> NotZeros = NotExpr::create(klee::ConstantExpr::create(Origin->KnownZeros, Width));
```

Comment:
don't do this work unless there are some nonzero bits, othewise you're going to be making queries larger in the common case where there's nothing there